edgedb/_testbase.py

```diff
@@ -594,6 +599,34 @@ def adapt_call(cls, result):
         return result
 
 
+class SQLATestCase(SyncQueryTestCase):
+    SQLAMODELS = None
+    DEFAULT_MODULE = 'default'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        class_set_up = os.environ.get('EDGEDB_TEST_CASES_SET_UP')
+        if not class_set_up:
+            # Now that the DB is setup, generate the SQLAlchemy models from it
+            spec = get_schema_json(cls.client)
+
+            with open(cls.SQLAMODELS, 'w') as f:
+                gen = ModelGenerator(out=f)
+                gen.render_models(spec)
```

Comment:
We should write the target module out into a temporary directory that we add to `sys.path`, instead of writing to files in the source tree

Reply:
It's a temp directory now.